tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -181,11 +189,20 @@ void
 InitializeDIFlags(const LL_IRFeatures *feature)
 {
 #ifdef FLANG_LLVM_EXTENSIONS
-  if (ll_feature_debug_info_ver70(feature)) {
+  DIFLAG_ARTIFICIAL = 1 << 6;
```

Comment:
It seems to me the DIFLAG_ARTIFICIAL = 1 << 6 should be pulled out of the FLANG_LLVM_EXTENSIONS guard.